src/Router.tsx

```diff
@@ -2,23 +2,43 @@ import { NetworkBar, Overlay } from "@govtechsg/tradetrust-ui-components";
 import React, { ReactElement } from "react";
 import { BrowserRouter, Route, Switch } from "react-router-dom";
 import { usePersistedConfigFile } from "./common/hook/usePersistedConfigFile";
-import { AddressResolverContainer } from "./components/AddressResolverContainer";
+import { SettingsContainer } from "./components/SettingsContainer";
 import { DynamicFormContainer } from "./components/DynamicFormContainer";
 import { FormSelectionContainer } from "./components/FormSelectionContainer";
 import { HomeContainer } from "./components/Home";
 import { PublishContainer } from "./components/PublishContainer";
+import { AddressBookContainer } from "./components/AddressBookContainer";
+import { AddressResolverContainer } from "./components/AddressResolverContainer";
+import { NavigationBar, NavigationBarProps } from "./components/NavigationBar";
+import { useConfigContext } from "./common/context/config";
+import { useFormsContext } from "./common/context/forms";
+
+const navigationBarProps: NavigationBarProps = {};
 
 export const Router = (): ReactElement => {
   const { configFile } = usePersistedConfigFile();
+  const { setConfig, config } = useConfigContext();
+  const { setForms, setActiveFormIndex } = useFormsContext();
+
+  const logout = (): void => {
+    setForms([]);
+    setActiveFormIndex(undefined);
+    setConfig(undefined);
+  };
+
+  if (config) {
+    navigationBarProps.logout = logout;
+  }
 
   return (
     <>
       <NetworkBar network={configFile?.network}>
         You are currently on <span className="capitalize">{configFile?.network}</span> network. To
         change it, please upload a new config file.
       </NetworkBar>
-      <main className="bg-blue-300">
-        <BrowserRouter>
+      <BrowserRouter>
+        <NavigationBar {...navigationBarProps} />
```

Comment:
y create another const to handle the navigationBarProps? can do something like this in the props?
```
logout={config ? logout : undefined}
or
config && logout={logout}
```

Reply:
right... wat was i thinking lol